src/Game.java

```diff
@@ -0,0 +1,114 @@
+package src;
+
+
+import java.util.Scanner;
+import java.util.Random;
+
+public class Game {
+    private Player player1;
+    private Player player2;
+
+    public Game(Player player1, Player player2) {
+        this.player1 = player1;
```

Comment:
Omdat je altijd null meegeeft aan de constructor hoef je de players niet te setten, want de standaardwaarde is toch null